src/netbase.cpp

```diff
@@ -749,3 +749,93 @@ void InterruptSocks5(bool interrupt)
 {
     interruptSocks5Recv = interrupt;
 }
+
+bool IsBadPort(uint16_t port)
+{
+    /* Don't forget to update doc/p2p-bad-ports.md if you change this list. */
```

Comment:
Not sure if this should be a doxygen comment. You did update the doxygen in the declaration :+1:  
```suggestion
    // Don't forget to update doc/p2p-bad-ports.md if you change this list.
```